test/sanity/basics/booleans/literals.wtest

```diff
@@ -0,0 +1,66 @@
+describe "booleans - tests with literals" {
+
+	test "true literal used in an assert" {
+		assert.that(true)
```

Comment:
Some files seem to be tabulated with tabs instead of the two spaces. Maybe we can run an automatic replacer before merging. I don't really mind, just pointing this out because I know you care XD

Reply:
Yeah, I did the job manually (a replacer in VSC), but then I added new files... good catch.

Reply:
I realized I also want to change Eclipse settings to conform a 2-space indentation (it's better in many places, when you are teaching, or pasting code in docs)